locales/zh-Hans/search.json

```diff
@@ -21,5 +21,6 @@
     "card_filtered": "从 {total} 张卡片中找到 {num} 张卡片。",
     "st_stamina": "所需体力 {s}",
     "st_ct": "CT {ct}",
-    "st_limit": "Live 中只生效 {l} 次"
+    "st_limit": "Live 中只生效 {l} 次",
+    "Use original strings": "仅在搜索时使用日文"
```

Comment:
* For better readability, could you move this string before all key-values whose key contains an underscore? That is to say, could you move this string before the line of `no_skip_sp_on_ctfilter`?
* Could you also add this to `en/search.json`:

``` jsonc
{
// ...,
  "Use original strings": "Use original strings",
// ...,
```

Reply:
Swapped strings order.
https://github.com/outloudvi/info-pride/blob/264df9b305328e91cd764ed1b7b36deea4d7796b/locales/zh-Hans/search.json#L16-L18

Added en string.
https://github.com/outloudvi/info-pride/blob/264df9b305328e91cd764ed1b7b36deea4d7796b/locales/en/search.json#L17